src/database/migrations/0003-create-applicants.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE applicants (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  external_id VARCHAR NOT NULL,
+  opted_in BOOLEAN NOT NULL DEFAULT false,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  UNIQUE(external_id)
```

Comment:
Earlier I was not sure about whether you had intended this to be unique within the PDC but here it is clear.